plugins/source/aws/resources/services/cloudtrail/channels.go

```diff
@@ -0,0 +1,70 @@
+package cloudtrail
+
+import (
+	"context"
+
+	"github.com/apache/arrow/go/v13/arrow"
+
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail"
+	"github.com/aws/aws-sdk-go-v2/service/cloudtrail/types"
+	"github.com/cloudquery/cloudquery/plugins/source/aws/client"
+	"github.com/cloudquery/plugin-sdk/v3/schema"
+	"github.com/cloudquery/plugin-sdk/v3/transformers"
+)
+
+func Channels() *schema.Table {
+	tableName := "aws_cloudtrail_channels"
+	return &schema.Table{
+		Name:                tableName,
+		Description:         `https://docs.aws.amazon.com/awscloudtrail/latest/APIReference/API_GetChannel.html`,
+		Resolver:            fetchCloudtrailChannels,
+		PreResourceResolver: getChannel,
+		Multiplex:           client.ServiceAccountRegionMultiplexer(tableName, "cloudtrail"),
+		Transform:           transformers.TransformWithStruct(&cloudtrail.GetChannelOutput{}, transformers.WithSkipFields("ResponseMetadata")),
+		IsIncremental:       true,
```

Comment:
```suggestion
		IsIncremental:       false,
```

I don't think this table is incremental?